clusters/app.ci/assets/admin_aos-qe-ci-to-qe-private-deck-ns_rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aos-qe-ci-to-qe-private-deck-ns
+  namespace: qe-private-deck
```

Comment:
Is this the namespace you meant? If so, it seems like this PR is doing 2 separate things. What are your goals with this change?

Reply:
I'm trying to grant aos-qe-ci rover group to access qe-private-deck.

Reply:
If that is the only goal of [clusters/app.ci/aos-qe-ci/admin_rbac.yaml](https://github.com/openshift/release/pull/39543/files#diff-383b30f7d859de570e69e01d96670fddc2cfd97e0065d304685f04da7cdfc0a4) is unnecessary.

Reply:
@smg247 Thanks for pointing it out. Updated now.